### DISCUSSION TITLE
macOS release

### DISCUSSION THREAD

**Discussion by pointlessone** [RubyistApp/Community#7] [General]
It'd be great to have a macOS version. Specifically to make Shortcuts that use Rubyist action on iOS usable on macOS.

**Comment by ApoB-100** [comment]
> It'd be great to have a macOS version. Specifically to make Shortcuts that use Rubyist action on iOS usable on macOS.

This would be really great if the current macOS restriction in the App Store for macOS could be lifted even if it's untested and at our own risk. 🙏

**Reply by morganism** [reply]
I've been looking for the same thing, the best workaround i can think of is the iOS sim in xcode
